src/main/java/com/epam/aidial/core/util/ProxyUtil.java

```diff
@@ -272,12 +266,12 @@ public static <T> T convertToObject(String payload, TypeReference<T> type) {
     }
 
     @Nullable
-    public static <T> T convertToObject(String payload, Class<T> clazz, boolean snakeCase) {
+    public static <T> T convertToObject(String payload, Class<T> clazz) {
```

Comment:
Did you replace the arg with Json annotations?

Reply:
Now snake case is defined with the annotation.